discv5/discv5-wire.md

```diff
@@ -100,7 +100,7 @@ In ordinary message packets and handshake message packets, the packet contains a
 authenticated message after the authdata section. For WHOAREYOU packets, the `message` is
 empty. Implementations must generate a unique `nonce` value for every packet.
 
-    message       = aesgcm_encrypt(initiator-key, nonce, message-pt, header)
+    message       = aesgcm_encrypt(initiator-key, nonce, message-pt)
```

Comment:
I was curious about this too.

Reply:
Assuming mitm attack: 
- for `ordinary` packet the only field which may be tweaked is `version`. Now there is nothing to do with it. But another version value would be possible there could be interesting cases
- for `handshake` packet almost all fields are signed with `id-signature` but there are still fields which may be tweaked. No ideas on a possible attack for now, but this change looks as security weakening

Reply:
@Nashatyrev for `handshake` MITM could swap ENR for an old version with an old host for example. Not clear what to do with packets on it after this without a private key, but looks not good.

Reply:
Hey, thanks for this analysis. I removed the authenticated data because seemed kind-of useless for regular message packets, but we can totally put it back!

Reply:
I'd vote for bringing it back cause without authorized header it looks shaky